pyne/r2s.py

```diff
@@ -9,6 +9,9 @@
 
 QA_warn(__name__)
 
+# source file version is composed of pyne version and one more int number
+_SOURCE_FILE_VERSION = [0, 7, 3, 1]
+_SOURCE_TAG_NAME = 'r2s_source_file_version'
```

Comment:
maybe `_SOURCE_VERSION_TAG_NAME` instead?

Reply:
Thanks @gonuke , changed as your suggestion in https://github.com/pyne/pyne/pull/1352/commits/b9e108dc6780089c33f39ac0b7f5ae75ed37b99c.